docs: What's New & Upgrade Guide 9.5

imatwawana opened the pull request:
This PR adds the What's New which describes new features and changes available in Grafana v9.5 and the 9.5 Upgrade Guide.

imatwawana committed:
added what's new and upgrade guide 9.5 and updated whats new index page

zsofiakm committed:
Added new nav and nested folders.

imatwawana committed:
added empty dashboard state and dashboards section

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,70 @@
+
+Also, a text Add dropdown with these options has replaced the previous icon at the top of the dashboard. This makes it clearer that this element allows you to not just add new panels, but to take all the actions associated with building a new dashboard.
+
+{{< figure src="/media/docs/grafana/screenshot-empty-dashboard-whats-new-9-5.png" max-width="750px" caption="Admin view of data source connection page on dashboard creation" >}}
```
```suggestion
{{< figure src="/media/docs/grafana/screenshot-empty-dashboard-whats-new-9-5.png" max-width="750px" caption="Dashboard without any visualizations added" >}}
```

imatwawana committed:
fixed caption

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,70 @@
+
+### Redesigned empty dashboard state
+
+_Generally available in all editions of Grafana using the emptyDashboardPage [feature toggle](https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/#feature_toggles). The default value is `true` and you can disable it in config._
+
+Dashboards have been updated so that it’s easier to begin building from an empty dashboard state. The options displayed when you add a new dashboard—adding a visualization, a row, or importing panels—each include brief explanations of what those steps will do, so you can begin building with confidence. 
+
+Also, a text Add dropdown with these options has replaced the previous icon at the top of the dashboard. This makes it clearer that this element allows you to not just add new panels, but to take all the actions associated with building a new dashboard.
```
```suggestion
_Generally available in all editions of Grafana using the emptyDashboardPage [feature toggle]({{< relref "../setup-grafana/configure-grafana/#feature_toggles" >}}). The default value is `true` and you can disable it in config._

Dashboards have been updated so that it’s easier to begin building from an empty dashboard state. The options displayed when you add a new dashboard—adding a visualization, a row, or importing panels—each include brief explanations of what those steps will do, so you can begin building with confidence. 

Also, a text **Add** dropdown with these options has replaced the previous icon at the top of the dashboard. This makes it clearer that this element allows you to not just add new panels, but to take all the actions associated with building a new dashboard.
```

imatwawana committed:
fixed formatting of link and UI element

natellium commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,70 @@
+
+### Redesigned empty dashboard state
+
+_Generally available in all editions of Grafana using the emptyDashboardPage [feature toggle]({{< relref "../setup-grafana/configure-grafana/#feature_toggles" >}}). The default value is `true` and you can disable it in config._
```
@imatwawana @polibb are we sure we want to mention the feature flag? This should be the default experience, when something is generally available we shouldn’t encourage people to disable it.

imatwawana replied:
I'll defer to @polibb on this, but IIRC, the point of the feature flag is mostly so that we have the option to roll back if needed. If that's still the case, then I agree, let's remove this.

natellium replied:
Yep, that’s why we have feature flags. They allow us to revert and do phased rollouts, it’s not for the users to change the default experience of Grafana once it’s stable :)

joshhunt commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,70 @@
+
+It's worth noting that the nested folders feature is currently experimental. As such, it's recommended to enable it only on test or development instances, rather than in production environments.
+
+To try out the nested folders feature, you'll need to enable the `nestedFolders` feature toggle. If you’re using Grafana Cloud, and would like to enable this feature, please contact customer support.
```
I'm hesitent to suggest people contact support for nested folders. i still think it's quite early and I don't want to create additional burden for cx

joshhunt replied:
I think my concern is around how nested folders is still in an exceptionally early and rough state with a lot of caveats, and it's less about "asking for a feature flag enabled is a burden" but "once customers enable this flag, they may have a bunch more requests", especially because we don't support disabling the flag once enabled.

GrafanaWriter replied:
For experimental features, customers are told explicitly that we do not provide support, as captured in [this documentation](https://grafana.com/docs/release-life-cycle/) we recently published.

eleijonmarck committed:
Add: onlyExternalOrgRoleSync

brendamuir committed:
Adds Alerting entries

brendamuir committed:
Adds alerting screenshots

brendamuir committed:
More alerting screenshots

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,130 @@
+
+Also, a text **Add** dropdown with these options has replaced the previous icon at the top of the dashboard. This makes it clearer that this element allows you to not just add new panels, but to take all the actions associated with building a new dashboard.
+
+{{< figure src="/media/docs/grafana/screenshot-empty-dashboard-whats-new-9-5.png" max-width="750px" caption="Dashboard without any visualizations added" >}}
```
```suggestion
{{< figure src="/media/docs/grafana/screenshot-empty-dashboard-whats-new-9-5.png" max-width="750px" caption="Dashboard without any visualizations added" >}}

### Dashboard panel redesign

_Generally available in all editions of Grafana._

Dashboard panels contain a lot of information, some of which is difficult to discover or access from the dashboard. With our redesigned panels, we've improved accessibility and made it easier to understand the status of a panel by adding and moving key elements.

We’ve improved the support of panels without a header, made a distinction between details set by you and data-induced information, and then included all essential components in the header of the panel. All of these are laid out from left to right in a row, so there are no overlapping, unusable components.

Grafana’s new panel is available only for React-based panels; no Angular-based panels are redesigned. For example, Angular-based panel will still have the old Graph and Table visualizations.
```

imatwawana committed:
add dashboard panel redesign

chri2547 committed:
Update _index.md

brendamuir committed:
Adds search to notification policy entry

brendamuir committed:
deletes old line

zsofiakm committed:
Removed nested folders (will be private preview and not part of what's new)

IevaVasiljeva committed:
Docs: support bundle section for 9.5 What's New (#65641)

* support bundle section for 9.5 what's new

* Update whats-new-in-v9-5.md

* Update docs/sources/whatsnew/whats-new-in-v9-5.md

Co-authored-by: Christopher Moyer <35463610+chri2547@users.noreply.github.com>

* doc feedback

---------

Co-authored-by: Eric Leijonmarck <eric.leijonmarck@gmail.com>
Co-authored-by: Christopher Moyer <35463610+chri2547@users.noreply.github.com>

itsmylife committed:
Influxdb database field deprecation notice

itsmylife committed:
Update reasoning

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,143 @@
+
+> **Note:** You must use relative references when linking to docs within the Grafana repo. Please do not use absolute URLs. For more information about relrefs, refer to [Links and references](/docs/writers-toolkit/writing-guide/references/).-->
+
+## Nav navigation
```
```suggestion
## New navigation
```

imatwawana committed:
fixed typo

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,143 @@
+
+Get additional help while configuring your Alertmanager. If you enter an invalid Alertmanager configuration, an error message displays, and you can choose from a previous working configuration to restart it.
+
+## InfluxDB provisioning change
+
+### Database field deprecation
```
@itsmylife - we want to avoid having two headings in a row. Can we simplify this like so:
```suggestion
## InfluxDB database field deprecation
```

eleijonmarck committed:
Docs: what's new API keys UI update (#65720)

* API keys deprecation

* Update docs/sources/whatsnew/whats-new-in-v9-5.md

Co-authored-by: Gabriel MABILLE <gamab@users.noreply.github.com>

* Update docs/sources/whatsnew/whats-new-in-v9-5.md

Co-authored-by: Gabriel MABILLE <gamab@users.noreply.github.com>

---------

Co-authored-by: Gabriel MABILLE <gamab@users.noreply.github.com>

eleijonmarck committed:
Docs: API keys UI updates (#65723)

update

ArturWierzbicki committed:
Docs: add dashboard previews deprecation notice (#65692)

* previews deprecation notice

* Apply suggestions from code review

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

---------

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

Commit:
we set the FF emptyDashboardPage ON by default, so it's available by default

itsmylife committed:
Update docs/sources/whatsnew/whats-new-in-v9-5.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

gtk-grafana committed:
add whats new toggle for prom browser cache feature

gtk-grafana committed:
Merge remote-tracking branch 'origin/main' into docs/whats-new-v9.5

bohandley committed:
Prometheus: metric encyclopedia feature toggle

bohandley committed:
spell check

mjseaman committed:
Product editorial pass

Adds hyperlinks, notes release stages, updates intro, Adds content to upgrade guide

mjseaman committed:
Merge branch 'docs/whats-new-v9.5' of github.com:grafana/grafana into docs/whats-new-v9.5

gtk-grafana committed:
add blurb for prom query cache

danielkenlee committed:
Add more context to org role sync

Add more context to organization role sync about the ability to change a user's organization role even then sync is enabled. Also outlined why this is valuable and when it should be used.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/grafana/grafana?pullRequest=64168) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/grafana/grafana?pullRequest=64168) before we can accept your contribution.<br/>**11** out of **12** committers have signed the CLA.<br/><br/>:white_check_mark: imatwawana<br/>:white_check_mark: zsofiakm<br/>:white_check_mark: chri2547<br/>:white_check_mark: IevaVasiljeva<br/>:white_check_mark: eleijonmarck<br/>:white_check_mark: ArturWierzbicki<br/>:white_check_mark: polibb<br/>:white_check_mark: itsmylife<br/>:white_check_mark: bohandley<br/>:white_check_mark: mjseaman<br/>:white_check_mark: gtk-grafana<br/>:x: danielkenlee<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/grafana/grafana?pullRequest=64168) it.</sub>

imatwawana committed:
copy edit of upgrade guide

imatwawana committed:
copy edit and link fixes

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,180 @@
+
+Welcome to Grafana 9.5! We're excited to share some major updates to Grafana's navigation, tons of usability improvements to Alerting, and some promising experiments to help you query your Prometheus metrics. Also, read on to learn about our continued migration from API keys to service accounts, as well as deprecation of plugins that use Angular and a field in the InfluxDB data source.
+
+For more detail about all the changes in this release, refer to the [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md). For the specific steps we recommend when you upgrade to v9.5, check out our [Upgrade Guide]({{< relref "../../upgrade-guide/upgrade-v9.5/" >}}).
```
@chri2547 - I can't get this link to the upgrade guide to work in my local. I'm not sure what I'm doing wrong here.

chri2547 replied:
Hm, tried a couple things, same result. @jdbaldry , what are we doing wrong here? Thank you!

chri2547 replied:
Fixed.

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,180 @@
+
+> **Note:** You must use relative references when linking to docs within the Grafana repo. Please do not use absolute URLs. For more information about relrefs, refer to [Links and references](/docs/writers-toolkit/writing-guide/references/).-->
+
+## Grafana's new navigation is GA
```
@chri2547 - What do we think about this heading? It feels awkward to me. I've made a suggestion below.
Whatever we decide here, I'll be applying to line 59 since that was also a GA.
```suggestion
## Grafana's new navigation is generally available
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,180 @@
+
+Navigate from an alert rule straight to a dashboard or a panel associated with the alert rule to visualize your alerting data.
+
+{{< figure src="/media/docs/alerting/dashboard-alert-rule.png" max-width="750px" caption="Access a dashboard or panel from an alert rule" >}}
```
@brendamuir - Is it possible to get this screenshot in dark mode to be consistent with all the other ones in the What's New?

brendamuir replied:
hi there-i've asked for it. most people are off today-hope to have it by tomorrow!

brendamuir replied:
Added!

imatwawana replied:
thank you!

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,180 @@
+## InfluxDB plugin database field deprecation
+
+The `database` field in the provisioning file has been deprecated.
+This information will be stored in `jsonData` with `dbName` field.
```
@itsmylife - Can we make this edit?

```suggestion
This information will be stored in the `jsonData` file using the `dbName` field.
```

itsmylife replied:
I will make that change like the following: 
```
This information will be stored in the `jsonData` field using the `dbName` property.
```

itsmylife replied:
Fixed here https://github.com/grafana/grafana/pull/64168/commits/9c6a4df961be5a6cac84009baf836e13a1b65b95

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,180 @@
+
+_Generally available in all editions of Grafana._
+
+Grafana v9.4 provided the ability to configure synchronization of organization roles for each OAuth provider. With synchronization on, the organization role was applied to the user from the OAuth provider upon signing in. After signing in, during the session, you could change the user’s organization role.
```
@danielkenlee - Does this edit work? Want to make this clearer.

```suggestion
Grafana v9.4 provided the ability to configure synchronization of organization roles for each OAuth provider. With synchronization on, the organization role was applied to the user from the OAuth provider upon signing in. However, after the user signed in, you could still change the user’s organization role during the session.
```

danielkenlee replied:
Much clearer, thanks Isabel!

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,180 @@
+
+The navigation in Grafana has been updated with a new design and an improved structure to make it easier for you to access the data you need. With this update, you can quickly navigate between pages, giving you full visibility into the health of your systems.
+
+As Grafana evolved from a visualization platform to a comprehensive observability solution, we added numerous tools to support users throughout the software development life cycle. These tools focus on preventing incidents, monitoring applications or infrastructure, and aiding in incident response. However, the added functionality needs to be easily discoverable and navigable for it to be truly h ...
+
+These are the key updates to Grafana’s navigation experience:
```
@zsofiakm - Does this edit work?

```suggestion
As Grafana evolved from a visualization platform to a comprehensive observability solution, we added numerous tools to support users throughout the software development life cycle. These tools focus on preventing incidents, monitoring applications or infrastructure, and aiding in incident response. However, the added functionality needs to be easily discoverable and navigable for it to be truly helpful. These key updates to Grafana’s navigation experience help address this:
```

danielkenlee committed:
Update docs/sources/whatsnew/whats-new-in-v9-5.md

Makes it clear what can happen after sign in.

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

chri2547 committed:
Update docs/sources/whatsnew/whats-new-in-v9-5.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

chri2547 committed:
Update docs/sources/whatsnew/whats-new-in-v9-5.md

Co-authored-by: Isabel <76437239+imatwawana@users.noreply.github.com>

chri2547 commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,178 @@
+
+Welcome to Grafana 9.5! We're excited to share some major updates to Grafana's navigation, tons of usability improvements to Alerting, and some promising experiments to help you query your Prometheus metrics. Also, read on to learn about our continued migration from API keys to service accounts, as well as deprecation of plugins that use Angular and a field in the InfluxDB data source.
+
+For more detail about all the changes in this release, refer to the [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md). For the specific steps we recommend when you upgrade to v9.5, check out our [Upgrade Guide]({{< relref "../../upgrade-guide/upgrade-v9.5/" >}}).
```
```suggestion
For more detail about all the changes in this release, refer to the [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md). For the specific steps we recommend when you upgrade to v9.5, check out our [Upgrade Guide]({{< relref "../../upgrade-guide/upgrade-v9.5/index.md" >}}).
```

chri2547 commented on docs/sources/upgrade-guide/upgrade-v9.5/index.md:
```diff
@@ -0,0 +1,31 @@
+  - documentation
+  - upgrade
+title: Upgrade to Grafana v9.5
+menutitle: Upgrade to v9.5
```
```suggestion
menuTitle: Upgrade to v9.5
```

chri2547 committed:
Apply suggestions from code review

chri2547 committed:
fixes link

brendamuir committed:
Adds dark screenshot for alerting

brendamuir committed:
fixes alerting title

frossano-grafana committed:
Update whats-new-in-v9-5.md

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,186 @@
+
+We updated the reporting UI to fit better the new navigation style, adding a horizontal slider and moving the Preview and Send buttons to the Action section in the page. We also fixed the alignment of the different sections. 
+
+{{< figure src="/media/docs/grafana/Screenshot-newUI-report.png" max-width="750px" caption="Grafana new report UI >}}
```
```suggestion
{{< figure src="/media/docs/grafana/Screenshot-newUI-report.png" max-width="750px" caption="Grafana new report UI" >}}
```

imatwawana committed:
added missing quotation mark

imatwawana committed:
trigger linting

imatwawana committed:
small change to trigger linting

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,186 @@
+
+{{< figure src="/media/docs/grafana/screenshot-empty-dashboard-whats-new-9-5.png" max-width="750px" caption="Dashboard without any visualizations added" >}}
+
+### Dashboard panel redesign
```
```suggestion
### Redesigned dashboard panel is generally available
```

imatwawana committed:
fix title to follow pattern

frossano-grafana committed:
Update whats-new-in-v9-5.md

frossano-grafana committed:
Merge branch 'docs/whats-new-v9.5' of https://github.com/grafana/grafana into docs/whats-new-v9.5

itsmylife committed:
Clearer explanation

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,192 @@
+
+_Experimental in all editions of Grafana, for on-prem users only._
+
+This feature adds support to use JWTs tokens to store rendering keys instead of relying on “remote caching”. It covers most of the rendering use cases, though some still relies on the remote cache as store. Users can enable it by enabling the feature flag `renderAuthJWT` in the custom.ini configuration. It allows to use JWT in rendering service instead of the main remote cache configured.
```
```suggestion
This feature adds support for using JWT tokens to store rendering keys instead of relying on “remote caching”. It covers most of the rendering use cases, though some still rely on the remote cache as a store. You can enable this by enabling the feature flag `renderAuthJWT` in the `custom.ini` configuration file. It allows you to use JWT in rendering service instead of the main remote cache configured.
```

imatwawana committed:
wording edits

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,192 @@
+
+{{< figure src="/media/docs/grafana/Screenshot-newUI-report.png" max-width="750px" caption="Grafana new report UI" >}}
+
+### Experimental support to use JWTs as auth method
```
```suggestion
## Experimental support to use JWTs as auth method
```

imatwawana committed:
fixed heading levels

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,192 @@
+
+{{< figure src="/media/docs/grafana/Screenshot-newUI-report.png" max-width="750px" caption="Grafana new report UI" >}}
+
+## Experimental support to use JWTs as auth method
```
```suggestion
## Support for using JWTs as auth method
```

imatwawana committed:
fix title wording

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,192 @@
+
+_Experimental in all editions of Grafana, for on-prem users only._
+
+This feature adds support for using JWT tokens to store rendering keys instead of relying on “remote caching”. It covers most of the rendering use cases, though some still rely on the remote cache as a store. You can enable this by enabling the feature flag `renderAuthJWT` in the `custom.ini` configuration file. It allows you to use JWT in rendering service instead of the main remote cache config ...
```
@frossano-grafana - is it okay for me to remove the last sentence? It seems like it's repetetive.

```suggestion
This feature adds support for using JWT tokens to store rendering keys instead of relying on “remote caching”. It covers most of the rendering use cases, though some still rely on the remote cache as a store. You can enable this by enabling the feature flag `renderAuthJWT` in the `custom.ini` configuration file. 
```

frossano-grafana replied:
Yes

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,192 @@
+
+## Experimental support for using JWTs as auth method
+
+_Experimental in all editions of Grafana, for on-prem users only._
```
@frossano-grafana - "All editions" should include Cloud. Is this only available in Enterprise and OSS?
```suggestion
_Experimental in all editions of Grafana, for on-prem users only._
```

frossano-grafana replied:
yes, only enterprise and OSS, I can change that

imatwawana replied:
I'll fix it. Thanks for the clarification!

imatwawana committed:
wording edit and availability fix

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,192 @@
+
+_Experimental in Grafana Open Source and Enterprise._
+
+This feature adds support for using JWT tokens to store rendering keys instead of relying on “remote caching”. It covers most of the rendering use cases, though some still rely on the remote cache as a store. You can enable this by enabling the feature flag `renderAuthJWT` in the `custom.ini` configuration file. 
```
```suggestion
This feature adds support for using JWT tokens to store rendering keys instead of relying on “remote caching”. It covers most of the rendering use cases, though some still rely on the remote cache as a store. You can enable this by enabling the feature flag `renderAuthJWT` in the `custom.ini` configuration file.
```

imatwawana committed:
remove extra space

imatwawana commented on docs/sources/upgrade-guide/upgrade-v9.5/index.md:
```diff
@@ -0,0 +1,31 @@
+
+### Migrate your API keys to service accounts
+
+We are upgrading Grafana [API keys]({{< relref "../../administration/api-keys/" >}}) to Service accounts. Service accounts are a superset of API Keys that include token rotation and compatibility with [Role-based access control]({{< relref "../../administration/roles-and-permissions/access-control/" >}}). In a future release, we'll automatically migrate all existing API keys to Service accounts.  ...
```
```suggestion
We are upgrading Grafana [API keys]({{< relref "../../administration/api-keys/" >}}) to service accounts. Service accounts are a superset of API Keys that include token rotation and compatibility with [Role-based access control]({{< relref "../../administration/roles-and-permissions/access-control/" >}}). In a future release, we'll automatically migrate all existing API keys to service accounts. All of your existing tokens will continue to work; they will simply be migrated to service accounts. You can preempt this change by migrating your existing API keys to Service accounts using Grafana's UI or API. Learn how to do this in the [API keys documentation]({{< relref "../../administration/api-keys/#migrate-api-keys-to-grafana-service-accounts" >}}).
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,192 @@
+
+## Removal of API key creation from the UI
+
+With this update we are going one step further in deprecating API Keys in favor of [service accounts]({{< relref "../administration/service-accounts/" >}}). We've removed the button for creating new API keys through Grafana's user interface, and now only allow the creation of API keys using our HTTP API. We recommend that you migrate your existing API keys to service accounts, and opt for new ser ...
```
```suggestion
With this update we are going one step further in deprecating API keys in favor of [service accounts]({{< relref "../administration/service-accounts/" >}}). We've removed the button for creating new API keys through Grafana's user interface, and now only allow the creation of API keys using our HTTP API. We recommend that you migrate your existing API keys to service accounts, and opt for new service accounts instead of new API keys. This change is part of our long-term strategy for sunsetting API keys.
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,192 @@
+
+We updated the reporting UI to better fit the new navigation style, adding a horizontal slider and moving the **Preview** and **Send** buttons to the Action section in the page. We also fixed the alignment of the different sections.
+
+{{< figure src="/media/docs/grafana/Screenshot-newUI-report.png" max-width="750px" caption="Grafana new report UI" >}}
```
```suggestion
{{< figure src="/media/docs/grafana/Screenshot-newUI-report.png" max-width="750px" caption="New Grafana report UI" >}}
```

imatwawana commented on docs/sources/whatsnew/whats-new-in-v9-5.md:
```diff
@@ -0,0 +1,192 @@
+
+_Experimental in Grafana Open Source and Enterprise._
+
+This feature adds support for using JWT tokens to store rendering keys instead of relying on “remote caching”. It covers most of the rendering use cases, though some still rely on the remote cache as a store. You can enable this by enabling the feature flag `renderAuthJWT` in the `custom.ini` configuration file.
```
```suggestion
This feature adds support for using JWT tokens to store rendering keys instead of relying on “remote caching”. It covers most rendering use cases, though some still rely on the remote cache as a store. You can enable this by enabling the feature flag `renderAuthJWT` in the `custom.ini` configuration file.
```

imatwawana committed:
final copy edit
